src/test/java/feature/uk/gov/ida/verifyserviceprovider/configuration/HubMetadataFeatureTest.java

```diff
@@ -67,6 +72,15 @@ public void setUp() {
             config("verifyHubConfiguration.metadata.trustStore.password", verifyHubKeystoreResource.getPassword())
         );
 
+        environmentHelper.setEnv(new HashMap<String, String>() {{
+            put("VERIFY_ENVIRONMENT", "COMPLIANCE_TOOL");
+            put("MSA_METADATA_URL", "some-msa-metadata-url");
+            put("MSA_ENTITY_ID", "some-msa-entity-id");
+            put("SERVICE_ENTITY_IDS", "[\"http://some-service-entity-id\"]");
+            put("SAML_SIGNING_KEY", TEST_RP_PRIVATE_SIGNING_KEY);
+            put("SAML_PRIMARY_ENCRYPTION_KEY", TEST_RP_PRIVATE_ENCRYPTION_KEY);
+        }});
```

Comment:
What is the reason for using environment variables here over config overrides? If it is to test that the env var substitutions work then that is fine but I think it is better to stay consistent otherwise.